pymobiledevice3/services/web_protocol/inspector_session.py

```diff
@@ -83,12 +94,12 @@ async def _receive_loop(self):
 
             response = self.protocol.inspector.wir_events.pop(0)
             response_method = response['method']
-            if response_method == 'Target.dispatchMessageFromTarget':
-                self._target_dispatch_message_from_target(response)
+            if response_method in self.response_methods:
+                self.response_methods[response_method](response)
             else:
-                error_message = 'Message is not supported'
+                error_message = 'Unknown response method'
                 logger.error(error_message, response)
-                raise NotImplementedError(error_message)
+                raise RuntimeError(error_message)
```

Comment:
`RuntimeError` is very implicit, better using `NotImplementedError` or a custom exception